canonical-web-and-design/www.canonical.com
# Update project directory

## Done

* Updated project directory for typos and updates

## QA

1. Check out this feature branch
2. Run the site using the command `./run`
3. View the site locally in your web browser at:http://0.0.0.0:8002/projects/directory
4. See that the text matches the [copy sheet](https://docs.google.com/spreadsheets/d/14Smfpq9ttrPt2jPx70NB3QTixH_6h7Ktnp1CihjT1So/edit#gid=0)
